File: 10_Switch/main.go
package main

import (
	"fmt"
	"strconv"
)

//create method to accept user input
func stringToInt(s string) int {
	i, err := strconv.Atoi(s)
	if err != nil {
		panic("Not a number")
	}
	return i
}

func main() {
	var numberString string
	fmt.Scanln(&numberString)
	number := stringToInt(numberString)

	switch number {
	case 1:
		fmt.Println("Sabelo")

	case 2:
		fmt.Println("Ntobeko")
	case 3:
		fmt.Println("Monwabisi")
	case 4:
		fmt.Println("Zukile")
	default:
		fmt.Printf("I have no idea what is %d", number)
	}
}

File: 10_Switch/main.py
def userInput():
    return int(input())

number = userInput()
if number == 1:
    print("Sabelo")
elif number ==2:
    print("Ntobeko")
elif number == 3:
    print("Monwabisi")
else:
    print("I have no idea what is %d" %number)

    #python3 main.py

# Alternative solution
number2 = int(input())
db = {1:"Sabelo", 2:"Ntobeko",3:"Monwabisi",4:"Zukile"}
print(db.get(number2,"I have no idea what is %d" %number))

#python3 main.py